src/drt/src/pa/FlexPA.cpp

```diff
@@ -178,6 +196,69 @@ void FlexPA::prep()
   prepPattern();
 }
 
+void FlexPA::prepPattern()
+{
+  ProfileTask profile("PA:pattern");
+
+  const auto& unique = unique_insts_.getUnique();
+
+  // revert access points to origin
+  unique_inst_patterns_.reserve(unique.size());
+
+  int cnt = 0;
+
+  omp_set_num_threads(router_cfg_->MAX_THREADS);
+  ThreadException exception;
+#pragma omp parallel for schedule(dynamic)
+  for (frInst* unique_inst : unique) {
+    try {
+      // only do for core and block cells
+      // TODO the above comment says "block cells" but that's not what the code
+      // does?
```

Comment:
I put this in a few years ago. Have we figured it out yet?

Reply:
I think the comment is just wrong and it is only std cells.   For blocks I don't think the code for instance and rows would work well.